dev/Deployment/DeploymentManager.h

```diff
@@ -3,18 +3,18 @@
 #pragma once
 #include <PackageInfo.h>
 #include <PackageDefinitions.h>
-#include "Microsoft.Windows.ApplicationModel.WindowsAppSDK.DeploymentManager.g.h"
+#include "Microsoft.Windows.ApplicationModel.WindowsAppRuntime.DeploymentManager.g.h"
 
-namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::implementation
+namespace winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::implementation
 {
     struct DeploymentManager
     {
         DeploymentManager() = default;
-        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::DeploymentStatus GetStatus();
-        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::DeploymentStatus GetStatus(hstring const& packageFullName);
-        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::DeploymentStatus Initialize();
-        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppSDK::DeploymentStatus Initialize(hstring const& packageFullName);
+        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentResult GetStatus();
+        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentResult Initialize();
     private:
+        static winrt::Microsoft::Windows::ApplicationModel::WindowsAppRuntime::DeploymentResult GetStatus(hstring const& packageFullName);
```

Comment:
Note - to reference the metadata type here, just say the last part of the namespace:

```c++
static WindowsAppRuntime::DeploymentResult GetStatus(hstring const& packageFullName);
static WindowsAppRuntime::DeploymentResult Initialize(hstring const& packageFullName);
```